README.md

```diff
@@ -24,6 +24,20 @@ You can also run the unit tests to do the above and ensure everything seems in o
 ### Fonts
 We use Proxima Nova fonts which are proprietary and cannot be committed to source control, see [fonts](https://github.com/duckduckgo/iOS/tree/develop/fonts/licensed). 
 
+## Debugging
+
+### Instruments
+
+We have Custom Instruments tool to help visualize and track events that happen during runtime.
+
+In order to run it:
+1. Build a Debug version and install it on Simulator/Device.
+2. Select Instruments target and run it on a Mac. New instance of Instruments app will be run that has a grayed out icon indicating that it works in debug mode with custom instruments attached.
+3. Select 'DDG Trace' template or setup a custom one by importing 'DDG Timeline' instrument from Library .
+5. Start recording.
```

Comment:
where is number 4? :)

I had to select the simulator and ddg app before starting recording, is that what number 4 is?

Reply:
Sounds like a good number 4 to me! 😄